cloud_chatops/lib/pr_reminder.py

```diff
@@ -16,20 +16,24 @@ class PostPRsToSlack:
     """
     This class handles the Slack posting.
     """
-    # The default channel is dev-chatops
-    def __init__(self, mention=False, channel="C06U37Y02R4"):
-        self.channel = channel
+
+    def __init__(self, mention=False):
+        self.channel = "C06U37Y02R4"  # STFC-cloud: dev-chatops
```

Comment:
why did you change this? - seems like it would be useful to change channel rather than hardcoding it. 
If you feel differently - perhaps setting a constant variable at the top of the file and referencing it here would be better?

Reply:
That is the default channel. Channel is taken on line 29 in the run function. I will set the variable how you mentioned